lib/Doctrine/DBAL/Connection.php

```diff
@@ -1971,6 +1971,19 @@ public function getSchemaManager()
         return $this->_schemaManager;
     }
 
+    /**
+     * Creates the SchemaManager that can be used to inspect or change
```

Comment:
I don't know if we are going to merge this up into 3.x, but in case we are, let's avoid unnecessary conflicts.

```suggestion
     * Creates a SchemaManager that can be used to inspect or change
```